druid/assets/logos/README.md

```diff
@@ -0,0 +1,19 @@
+# Logos
```

Comment:
This file should get removed once the logos are in?

Reply:
Yes, I think it make sense to not keep it.